### ISSUE TITLE
[Bug] [iOS] Recipe gets read aloud twice

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#3732]
### Description

Recipe name is read aloud twice from search hit results. This should be fixed so that the information is not redundant and it's only read aloud once.

### Steps to Reproduce

1. Run recipes app on iOS
2. Activate VoiceOver
3. Click on Keto button
4. Swipe to first search result "Keto Fat Bombs"
2. Listen "Keto Fat Bombs, Button, Dessert, Keto Fat Bombs" is read aloud. The recipe name is read out twice

### Version with bug

Preview 11

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15

**Comment by rachelkang** [comment]
This seems to be happening due to [iOS Text Recognition](https://support.apple.com/en-us/HT211899) so we have to figure out how to disable it. Perhaps the recipes API is passing this information somehow

> When Text Recognition is on, VoiceOver reads the text it identifies within images and photos.

Additionally, random phrases related to the recipe item (that are noted nowhere) get announced. I thought this may be due to similar iOS system automated descriptions, but not sure what's going on. (For example, the second keto recipe says something about "chocolate chip". We see chocolate chips in the photo but it's not actually written anywhere.)

Anyways, closing this for now. I made it match Android by remove the image from the accessibility tree so it'll just read out the label that's overlaid on top, and avoids all these problems
